remove dead code for suggesting legacy dynamic shapes fixes

avikchaudhuri opened the pull request:
Summary: `dynamic_dim` based dynamic shapes are long gone, so pretty-printing suggested fixes for them is dead code.

Test Plan: existing tests

Differential Revision: D61398303

pianpwk commented:
Does this work well with this test case? I remember running into some problems with it: https://github.com/pytorch/pytorch/blob/1df1d00ffce9a90b885dbb0faf76a40030565320/test/test_dynamic_shapes.py#L1617

avikchaudhuri committed:
remove dead code for suggesting legacy dynamic shapes fixes (#133700)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/133700

`dynamic_dim` based dynamic shapes are long gone, so pretty-printing suggested fixes for them is dead code.

Test Plan: existing tests

Reviewed By: zhxchen17

Differential Revision: D61398303
